app/helpers/hanuman/application_helper.rb

```diff
@@ -39,7 +39,7 @@ def sortable(column, title, path_call_string = nil)
       if path_call_string
         link_to raw(title), eval(path_call_string+"(params.merge(:sort => column.gsub('*,', '!').gsub(',', ' ' + direction + ',').gsub('*', ' asc').gsub('!', ' asc,'), :direction => direction, :page => nil))"), {:class => css_class}
       else
-        link_to raw(title), params.merge(:sort => column.gsub('*,', '!').gsub(',', ' ' + direction + ',').gsub('*', ' asc').gsub('!', ' asc,'), :direction => direction, :page => nil), {:class => css_class}
```

Comment:
We updated this line due to this error we were getting during smoke testing:

```
ActionView::Template::Error (Attempting to generate a URL from non-sanitized request parameters! An attacker can inject malicious data into the generated URL, such as changing the host. Whitelist and sanitize passed parameters to be secure.)
```

Using `request.params` maintains the old behavior which assumes all the params are safe by default. This could be a potential security risk. Is `sortable` ever called with user provided input in its arguments? If so, let’s determine which params should be allowed, if not then this should be safe as is.

Reply:
Another option is we can also decide on a list of keys that are allowed for this method. That way we can keep on using `request.params`